tools/packaging/packaging.psm1

```diff
@@ -886,6 +742,324 @@ function New-UnixPackage {
     }
 }
 
+
+function Get-FpmArguments
+{
+    param(
+        [Parameter(Mandatory,HelpMessage='Package Name')]
+        [String]$Name,
+
+        [Parameter(Mandatory,HelpMessage='Package Version')]
+        [String]$Version,
+
+        [Parameter(Mandatory)]
+        [String]$Iteration,
+
+        [Parameter(Mandatory,HelpMessage='Package description')]
+        [String]$Description,
+
+        [Parameter(Mandatory,HelpMessage='Installer Type')]
+        [String]$Type,
```

Comment:
Please add a comment for the possibilities.

Reply:
resolved